packages/amplify-console-integration/src/util.ts

```diff
@@ -22,7 +22,7 @@ export function getCLIPath() {
 }
 
 export function getProfileName() {
-  return 'amplify-console-integration-test';
+  return 'amplify-integ-test-user';
```

Comment:
Shouldn't this be "default"?

Reply:
I'm just using the same profile name we are using for all the other e2e tests. 
And it does not matter which string we use, the profile will be created if it does not existing. 
Since we don't want to create another profile, I'm just using the same name that the other tests are using. 
check the amplify-e2e-tests package

Reply:
In CI amplify-integ-test-user is always the profile name configured, this is right